WES+WGS/Pipeline.nf

```diff
@@ -11,14 +11,13 @@
 process HarmonizeSM {
    cache "lenient"
    cpus 1
-   memory "4 GB"
+   memory "1 GB"
```

Comment:
This applies only for your situation and should not be recommended as default

Reply:
reverted